### PR TITLE
Add DataDog RUM init to decision review templates

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > We want to add DataDog Real User Monitoring session to this list of static pages (in impact section). This PR updates the liquid template for each template and adds a new includes file which initializes RUM sessions on the page. This PR will be initially opened in draft mode to allow updates to the variable & add feature toggles. The RUM initialization code is also incomplete, and will be provided once we get data on traffic and percentage of sessions to record.
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_
  > _Pending_

## Related issue(s)

[See #83078](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83078)

## Testing done

- _Describe what the old behavior was prior to the change_
  > No DataDog RUM session recording, only Google Analytics
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

N/A - no visual changes

## What areas of the site does it impact?

Authenticated & Unauthenticated

| EntityId | Template | Path |
|---|---|---|
| 3071  | [landing_page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/landing_page.drupal.liquid) | decision-reviews |
| 3011  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/supplemental-claim |
| 3003  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/higher-level-review |
| 3029  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/board-appeal |
| 3010  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/board-appeal/veterans-law-judge-hearing |
| 3013  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/board-appeal/after-board-appeal-decision |
| 3008  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/after-you-request-review |
| 3030  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/contested-claims |
| 3025  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/insurance-claims |
| 3016  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/fiduciary-claims |
| 3102  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/legacy-appeals |
| 3103  | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/legacy-appeals/priority-review |
| 57215 | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/clinical-appeals |
| 57308 | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | decision-reviews/family-caregiver-program-reviews |
| 62667 | [page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/page.drupal.liquid) | get-help-with-review-request |
| 23290 | [support_resources_detail_page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/support_resources_detail_page.drupal.liquid) | resources/choosing-a-decision-review-option |
| 50010 | [support_resources_detail_page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/support_resources_detail_page.drupal.liquid) | resources/decision-reviews-faqs |
| 64063 | [support_resources_detail_page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/support_resources_detail_page.drupal.liquid) | resources/hearing-coordinators-for-the-board-of-veterans-appeals |
| 8520 | [step_by_step.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/step_by_step.drupal.liquid) | resources/how-to-check-your-va-claim-appeal-or-decision-review-status-online |
| 53778 | [support_resources_detail_page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/support_resources_detail_page.drupal.liquid) | resources/requesting-a-virtual-hearing-for-a-board-appeal |
| 49994 | [support_resources_detail_page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/support_resources_detail_page.drupal.liquid) | resources/vas-duty-to-assist |
| 9626 | [support_resources_detail_page.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/support_resources_detail_page.drupal.liquid) | resources/what-your-decision-review-or-appeal-status-means |

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/site/includes/decision-reviews-rum.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<script
+  nonce="**CSP_NONCE**"
```
Comment (shaunburdick) [inline src/site/includes/decision-reviews-rum.html:2]: Is this a real value?

**Review by shaunburdick — APPROVED** [review]
**LGTM**

File: src/site/includes/decision-reviews-rum.html (hunk 1 above)
Comment (dsasser) [reply src/site/includes/decision-reviews-rum.html:2]: This is a placeholder that [gets replaced](https://depo-platform-documentation.scrollhelp.site/developer-docs/content-security-policy#:~:text=nonce%C2%A0header%20value%20and%20HTML%20attribute%3A%20A%20unique%20value%20(nonce)%20is%20generated%20for%20each%20page%20visit.%20The%20nonce%20is%20added%20to%20the%20CSP%20header%20and%20to%20each%20occurance%20of%20the%20nonce%20attribute%20in%20the%20HTML%20document.%20The%20nonce%20is%20added%20to%20the%20HTML%20document%20with%20a%20simple%20string%20replacement%20of%C2%A0CSP_NONCE.) on-the-fly during page requests by the server response.

**Review by dsasser — APPROVED** [review]
Looks great!

I tested locally and the rum script is being included on intended pages, eg:
Screenshot 2024-07-30 at 11 11 10 AM
Screenshot 2024-07-30 at 11 10 19 AM

**Pull request merged**